### DISCUSSION TITLE
因應後疫情新常態，Appier 提 MarTech 發展三大主旋律

### DISCUSSION THREAD

**Discussion by Yilin20021016** [1101-GS3073-C/Python-Community-GS3073-C#552] [General]
Appier 沛星互動科技自 2012 年創立，在 10 月 20 日已屆滿 10 週年，營運據點擴及全球 17 個城市，包括東北亞的東京、大阪、首爾以及北美的舊金山、歐洲的巴黎等，為超過 1,300 個全球知名品牌與代理服務。針對快速演進的數位行銷市場，Appier 也提出因應 2023 年新常態的 MarTech 三大主旋律，以圍繞第一方數據的解決方案、線上線下數據的虛實整合與應用、以及強調轉換與成效，以結果為導向的行銷側重，助力企業在高度不確定的市場上，透過 AI 持續推進業務的成長。
像是以 Cookie 等收集的第三方數據即將退場，Appier 持續精進 AI 演算法與模型的表現，運用創新產品和技術，協助客戶在有限的數據與資源下，加強行銷的有效性。
https://technews.tw/2022/12/02/appier-customer-day-2022/#more-995722

**Comment by Ben6uo** [comment]
哇 看來以後會有更多更新的演算法出來

**Comment by williamyen0623** [comment]
先是IE 接著是cookie, 能親眼見證時代更替真是令人興奮
